app/api/utils/specs/__snapshots__/handleError.spec.js.snap

```diff
@@ -12,13 +12,14 @@ test error",
 exports[`handleError when the body contains the user and password should not show them in the log 1`] = `
 Array [
   Array [
-    Object {
-      "pretty": "
+    "
 body: {
  \\"username\\": \\"########\\",
  \\"password\\": \\"########\\"
 }
 test error",
+    Object {
+      "shouldBeMultiTenantContext": true,
```

Comment:
@RafaPolit why do we include this info on the error ? this is supposed to be internal? or do you think its going to be useful for debugging ?

Reply:
The handle error is bubbling up this into the transport in order to add the tenant.  That is required, I think (and it was already there for the error log, I just combined it into the debug call.

The final error is neither reporting this nor logging this into the files.  Do you think that is good enough?